Framework/script/RepoCleaner/rules/last_only.py

```diff
@@ -34,24 +36,32 @@ def process(ccdb: Ccdb, object_path: str, delay: int, extra_params: Dict[str, st
         if earliest == None or v.validFromAsDt > earliest.validFromAsDt:
             earliest = v
     logger.debug(f"earliest : {earliest}")
-            
+
+    logger.debug(f"{from_timestamp}")
+    logger.debug(f"{to_timestamp}")
```

Comment:
if you want to keep it, then please explain what will the two numbers mean in the logs.

Reply:
removed